src/mxml.mk

```diff
@@ -4,16 +4,16 @@ PKG             := mxml
 $(PKG)_WEBSITE  := https://michaelrsweet.github.io/
 $(PKG)_DESCR    := Mini-XML
 $(PKG)_IGNORE   :=
-$(PKG)_VERSION  := 2.9
-$(PKG)_CHECKSUM := cded54653c584b24c4a78a7fa1b3b4377d49ac4f451ddf170ebbc8161d85ff92
+$(PKG)_VERSION  := 2.10
+$(PKG)_CHECKSUM := 267ff58b64ddc767170d71dab0c729c06f45e1df9a9b6f75180b564f09767891
 $(PKG)_SUBDIR   := $(PKG)-$($(PKG)_VERSION)
 $(PKG)_FILE     := $(PKG)-$($(PKG)_VERSION).tar.gz
 $(PKG)_URL      := https://github.com/michaelrsweet/mxml/releases/download/release-$($(PKG)_VERSION)/$($(PKG)_FILE)
```

Comment:
And how to use it for downloading tarballs prepared by upstream instead of automatically generated tarballs for git tags?

```
$ grep _GH_CONF src/*.mk | wc -l
8
```
It is not very helpful...